cast/src/main.rs

```diff
@@ -76,13 +79,36 @@ async fn main() -> Result<()> {
             eprintln!("Transaction hash: {}", declared_contract.transaction_hash);
             Ok(())
         }
-        Commands::Call(args) => {
-            let block_id = get_block_id(&args.block_id).unwrap();
+        Commands::Deploy(deploy) => {
+            let account = get_account(&cli.account, &cli.accounts_file_path, &provider, &network)?;
```

Comment:
get_account returns `&mut account`, are your sure there are no issues with this?
we can change what it returns, as right now nothing actually uses mut (we can change it back in the future if needed)

Reply:
I think there are no issues, but also I don't know if there can be some. Could you elaborate on such cases?

Reply:
nvm, account will no longer be mut https://github.com/software-mansion/protostar/pull/2126